test/generators/install_generator_test.rb

```diff
@@ -19,7 +19,7 @@ def copy_directory(dir)
   def set_test_environment
     VCR.configure do |c|
       c.cassette_library_dir = 'test/fixtures/vcr_cassettes'
-      c.hook_into :webmock # or :fakeweb
+      c.hook_into :faraday # or :webmock or :fakeweb
```

Comment:
Hmm, perhaps it seems a webmock issue on jRuby.
However I'm not sure why the following tests are failed.
https://travis-ci.org/emberjs/ember-rails/jobs/363684591